package.json

```diff
@@ -62,13 +62,13 @@
     "coffeescript": "~1.12.7",
     "dotenv": "^4.0.0",
     "dtslint": "^1.0.3",
-    "gulp": "^3.9.1",
-    "gulp-coffee": "^2.3.5",
+    "gulp": "^4.0.2",
+    "gulp-coffee": "^3.0.3",
     "gulp-coffeelint": "^0.6.0",
     "gulp-insert": "^0.5.0",
-    "gulp-mocha": "^4.3.1",
+    "gulp-mocha": "^7.0.2",
```

Comment:
Needs to be 6.x as 7.x drops node 6 support (which is what the appveyor tests run with)